litellm/utils.py

```diff
@@ -6620,7 +6620,7 @@ def _get_max_position_embeddings(model_name):
             raise Exception()
     except:
         raise Exception(
-            "This model isn't mapped yet. Add it here - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json"
+            f"Model {model} from provider {custom_llm_provider} isn't mapped yet. Add it here - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json"
```

Comment:
@kmheckel this would cause another error - if custom_llm_provider isn't found from 'get_llm_provider'

Reply:
I'll remove the custom_llm_provider reference in that case.